CMakeLists.txt

```diff
@@ -110,6 +110,11 @@ endif()
 
 # Android options
 if(SFML_OS_ANDROID)
+    sfml_set_option(SFML_ANDROID_USE_SUSPEND_AWARE_CLOCK FALSE BOOL "TRUE to use the suspend-aware sf::Clock implementation, FALSE to default to another available monotonic clock")
```

Comment:
Suspend-aware means it continues running or stops during activity suspensions? Maybe mention this instead, the term "suspend-aware" is already in the variable name 🙂

I don't know what a good default here is, depends a bit on how the developers use the clock. On Desktop environments, the clock keeps running even if the user loses window focus; but then again mobile has so many different paradigms.

Reply:
It takes suspension time into account once the system goes into sleep. I'll think of a more descriptive doc here 👍 

Personally, a good default is the one consistent across platforms. Any monotonic clock with a microsecond precision is good enough. Suspend-awareness feels more like a gimmick. Android should provide a separate interface for observers working in the background. I am not an Android developer so I don't know but I would expect such a thing.